pkgs/development/ocaml-modules/tezos-bls12-381-polynomial/default.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  fetchFromGitLab,
+  buildDunePackage,
+  bls12-381,
+  data-encoding,
+  mec,
+  alcotest,
+  alcotest-lwt,
+  bisect_ppx,
+  qcheck-alcotest,
+}:
+
+buildDunePackage rec {
+  pname = "tezos-bls12-381-polynomial";
+  version = "0.1.2";
+  duneVersion = "3";
+  src = fetchFromGitLab {
+    owner = "nomadic-labs/cryptography";
+    repo = "privacy-team";
+    rev = "v${version}";
+    sha256 = "sha256-HVeKZCPBRJWQXkcI2J7Fl4qGviYLD5x+4W4pAY/W4jA=";
+  };
+
+  propagatedBuildInputs = [bls12-381 data-encoding mec];
```

```suggestion
  propagatedBuildInputs = [bls12-381 data-encoding];
```

Library `mec` is not needed, is it?

---

You are correct, fixed both review comments